tst/com/amazon/corretto/crypto/provider/test/EcGenTest.java

```diff
@@ -52,6 +54,14 @@
 @ResourceLock(value = TestUtil.RESOURCE_GLOBAL, mode = ResourceAccessMode.READ)
 public class EcGenTest {
     public static final String[][] KNOWN_CURVES = new String[][] {
+            // Not supported by Openssl
```

Comment:
Let's remove this comment. It is old and I'm not sure what it originally intended. (Same with other copy of this comment elsewhere in the same file.)